mappings/net/minecraft/block/entity/AbstractFurnaceBlockEntity.mapping

```diff
@@ -1,13 +1,23 @@
 CLASS net/minecraft/unmapped/C_dlxbwxyf net/minecraft/block/entity/AbstractFurnaceBlockEntity
 	FIELD f_apovtvbw SIDE_SLOTS [I
+	FIELD f_bitqkham BURN_TIME_STANDARD I
+	FIELD f_cjviljdx DATA_FUEL_TIME I
+	FIELD f_debmtawn RESULT_SLOT I
 	FIELD f_dswenqhr cookTimeTotal I
+	FIELD f_erasrdjo FUEL_SLOT I
 	FIELD f_fwsjbtjw TOP_SLOTS [I
+	FIELD f_hpqapnfb BURN_COOL_SPEED I
```

Comment:
This is vague and seems like a type (I know its not) Could it be clarified?

Reply:
I think "decay" would be appropriate here, perhaps 
```suggestion
	FIELD f_hpqapnfb BURN_DECAY I
```
or
```suggestion
	FIELD f_hpqapnfb BURN_DECAY_RATE I
```
?